### PR TITLE
add schema for validating meta field of assets

### PR DESCRIPTION
**Pull request by iamsdas**
fixes: #799

### CODE REVIEW

**Comment by iamsdas** [comment]
We will be needing another migration to handle the existing invalid meta objects. Before I work on that, can someone confirm the correctness of this implementation?

**Comment by vigneshhari** [comment]
ignore the existing values since we cant really do anything in those cases.

File: care/facility/models/asset.py [hunk 1]
```diff
@@ -89,6 +91,10 @@ class Meta:
     def __str__(self):
         return self.name
 
+    def save(self, *args, **kwargs) -> None:
+        self.clean_fields()
```
Comment (vigneshhari) [inline care/facility/models/asset.py:95]: Why is this required? also where is this defined?

Comment (iamsdas) [reply care/facility/models/asset.py:95]: The validators defined using `validators=[JSONFieldSchemaValidator(ASSET_META)]` in the model fields are not run by default while saving the model. So to enforce the schema, we need to manually call these validators. This is done using the `clean_fields` method (it's inbuilt in the Django ORM).

Comment (vigneshhari) [reply care/facility/models/asset.py:95]: We did not require this in the daily round models, not sure about this thou.

Comment (iamsdas) [reply care/facility/models/asset.py:95]: It turns out this validation is handled by the `ModelSerializer`. So all the changes made using the API calls should be validated and only the direct/non-api changes won't be validated. I hadn't run the migration initially while testing, making me overthink the issue.

So yeah, we don't really need it.

**Review by vigneshhari — APPROVED** [review]
